idaes/apps/caprese/examples/cstr_rodrigo/mhe_main_give_variance.py

```diff
@@ -0,0 +1,186 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2019, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Example for Caprese's module for MHE.
+"""
+import random
+from idaes.apps.caprese.dynamic_builder import DynamicSim
+from idaes.apps.caprese.util import apply_noise_with_bounds
+from pyomo.environ import SolverFactory, Reference
+from pyomo.dae.initialization import solve_consistent_initial_conditions
+# import idaes.logger as idaeslog
+from cstr_rodrigo_model import make_model
```

Comment:
These "implicit local imports" will fail if this file is imported from from outside of this directory. Here and elsewhere, instead, you should do something like `from idaes.apps.caprese.examples.cstr_rodrigo.cstr_rodrigo_model import make_model`.